scalameta/scalameta/shared/src/test/scala/scala/meta/tests/inputs/FormatMessageSuite.scala

```diff
@@ -22,7 +22,7 @@ class FormatMessageSuite extends FunSuite {
     |<input>:1: error: foo
     |
     |^
-  """.trim.stripMargin)
+  """.trim.stripMargin.split('\n').mkString(EOL))
```

Comment:
Would it make sense to move this to `.multiline` extension method that can be reused for all tests?

Reply:
Maybe factor out the entire comparison to something like `sameLines`?

Reply:
Agreed, that is even better ^^ 👍